src/components/ProductDetail.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+    <NavBar/>
+    <main class="flex justify-center">
+       
+		<div id="details" class="details p-5 flex mt-28 w-[900px] border sm:flex-col sm:mx-44">
+            <div class="relative mx-5">
+                <div>
+                    <div class="flex">
+                        <swiper
+                            :modules="modules"
+                            :slides-per-view="1"
+                            :space-between="50"
+                            navigation
+                            :pagination="{ clickable: true }"
+                            class="h-80 w-96"  
+                        >
+                            <swiper-slide v-for="(pro,index) in productList.images" :key="index"><img :src="pro" class="h-80 w-96"/></swiper-slide>
+                            
+                        </swiper>
```

Comment:
instead of using thirdpart library for swiper.. I think you have already created swiper using plain js.So you can also create your own swiper component.for now

Reply:
done

Reply:
ok done